sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsMeta.scala

```diff
@@ -810,6 +810,19 @@ abstract class AggExprMeta[INPUT <: AggregateFunction](
   def convertToGpu(child: Expression): GpuExpression
 }
 
+abstract class ImperativeAggExprMeta[INPUT <: ImperativeAggregate](
+    expr: INPUT,
+    conf: RapidsConf,
+    parent: Option[RapidsMeta[_, _, _]],
+    rule: DataFromReplacementRule)
+    extends ExprMeta[INPUT](expr, conf, parent, rule) {
+
+  override final def convertToGpu(): GpuExpression =
+    convertToGpu(childExprs(0).convertToGpu(),childExprs(1).convertToGpu())
```

Comment:
@nartal1 can you double check that all `ImperativeAggregate`s have two children?

Reply:
Thanks @abellina  for catching this. All ImperativeAggregates don't have two children. I have updated the code.